.env

```diff
@@ -1 +1,5 @@
 ELK_VERSION=6.4.0
+
+# Check Search Guard Versions Metrix https://docs.search-guard.com/latest/search-guard-versions
+SG_VERSION=6.4.2-23.1
```

Comment:
I would remove the ELK version and keep only the Search Guard version suffix (23.1 and 15). The full version number can be easily assembled from it inside the Dockerfile.

Reply:
```suggestion
SG_VERSION=23.1
```